.github/workflows/_e2e_tests.yaml

```diff
@@ -42,6 +42,12 @@ jobs:
         with:
           fetch-depth: 0
 
+      - id: setup_golang
+        name: setup golang
+        uses: actions/setup-go@v4
+        with:
+          go-version-file: go.mod
+
```

Comment:
Changed this because the default version in `ubuntu-latest` does not support `toolchain` in `go.mod`.